services/ui-src/.env_example

```diff
@@ -4,7 +4,7 @@ APPLICATION_ENDPOINT=https://mdctmcrdev.cms.gov/
 COGNITO_IDENTITY_POOL_ID=us-east-1:76708bb0-a458-4ea7-b90e-995ff5da5ab6
 COGNITO_IDP_NAME=OktaOIDC
 COGNITO_REDIRECT_SIGNIN=http://localhost:3000/
-COGNITO_REDIRECT_SIGNOUT=http://localhost:3000/
```

Comment:
I don't think `COGNITO_REDIRECT_SIGNOUT=http://localhost:3000/` needs to be removed, just adding the new one should do the trick

Reply:
Thanks, @BearHanded. It should be updated now.